lyte_helper.php_example

```diff
@@ -110,4 +110,11 @@ function fgo_lyte_change_css($lyte_css) {
 function no_lyte_on_mobile(){
 	return false;
 	}
+
+/** use internal lazy loader if you don't have another one configured on your site; the default for this filter is true */
+//add_filter('lyte_use_internal_lazyloader','__return_false');
```

Comment:
I would like the default to be `false` for now (to keep LYTE functioning identically as it is now), we can change to `true` later if all systems are go?

Reply:
Right, I changed in the latest commit.